components/nav/NavFooter.vue

```diff
@@ -34,7 +34,7 @@ function toggleDark() {
       </CommonTooltip>
     </div>
     <div>
-      <NuxtLink href="https://mozilla.social/about" target="_blank" external>
+      <NuxtLink to="/about" target="_blank">
```

Comment:
Nit: Can we remove the `target="_blank"`? I don't really see a reason for it to open in a new tab anymore since it doesn't leave Elk

Not a blocker or anything though

Reply:
Also that would be consistent with the Settings link